build.gradle.kts

```diff
@@ -376,6 +377,8 @@ project(":toml") {
     dependencies {
         implementation(project(":"))
         implementation(project(":common"))
+        implementation("org.eclipse.jgit:org.eclipse.jgit:5.9.0.202009080501-r") { exclude("org.slf4j") }
+        implementation("com.google.code.gson:gson:2.8.6")
```

Comment:
The platform has dependency jars of gson and jackson libraries so I don't think we should add our own version of gson into plugin